### DISCUSSION TITLE
Encoding and Keeping Subtitles

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#1864] [General]
How do I encode videos to lossless formats, as well as gif and apng files? Does LosslessCut preserve embedded subtitles? If not, what other video cutter can do these things?

**Comment by mifi** [comment]
> How do I encode videos to lossless formats, as well as gif and apng files?

losslesscut cannot do that, see #372

> Does LosslessCut preserve embedded subtitles

probably if the conversion is compatible.
